packages/api-contract/src/types.ts

```diff
@@ -68,12 +96,69 @@ export interface ContractABIFn extends ContractABIMeta {
   (...args: CodecArg[]): Uint8Array;
 }
 
+// export interface ContractABIArgV2 {
```

Comment:
We should rather forgo the commented out types - still marked WIP, so still ok, but something to keep in mind and just clean up before the flag is removed.